registry-api/generate-typescript.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+CURRENT_DIR=$(pwd)
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+CODEGEN_JAR=~/.ivy2/cache/io.swagger/swagger-codegen-cli/jars/
```

Comment:
Looks like it's still needed.  The change just avoids a "no implicit any" error from the typescript compiler.  Here's the change:
https://github.com/swagger-api/swagger-codegen/compare/master...TerriaJS:magda

I'm not 100% sure, though, because `generate-typescript.sh` is just spewing errors on my system at the moment. ☹️